python/src/main/python/drivers/runspv.py

```diff
@@ -327,21 +327,37 @@ def remove_end(str_in: str, str_end: str):
     return str_in[:-len(str_end)]
 
 
+# Runs spirv-opt on |spv_file| with arguments |spirv_opt_args|.  The optimized binary file is
```

Comment:
It is more common to use docstrings: """ """ within the function body.

Reply:
Done.